src/tools/tidy/src/deps.rs

```diff
@@ -40,6 +40,7 @@ const EXCEPTIONS: &[(&str, &str)] = &[
     ("snap", "BSD-3-Clause"),    // rustc
     // FIXME: this dependency violates the documentation comment above:
     ("fortanix-sgx-abi", "MPL-2.0"), // libstd but only for `sgx` target
+    ("libloading", "ISC"),           // MIT-like
```

Comment:
The new license exception needs attention, but I'm not sure from whom exactly.
cc @Mark-Simulacrum

Reply:
AFAIK MIT and ISC are considered functionally equivalent, but I am not a lawyer. https://en.wikipedia.org/wiki/ISC_license

Reply:
I can sign off on using ISC as a library exception as a member of core. I did consult the foundations lawyer before.

Reply:
We should also add ISC to the LICENSES list (i.e., it can always be used). Thanks for the ping!